### PR TITLE
Add release information

### LINKED ISSUE #75: Add information how to run releases
We should offer information how to run a compiled release and promote it to the standard way to install matterpoll

### PR DESCRIPTION
**Pull request by Hanzei**
Please check spelling.
Fixes #75

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,25 +18,54 @@ Polling feature for Mattermost's custom slash command.
...
+./matterpoll-emoji
+```
+
+#### Compile the source my yourself
+
+Clone this repository and checkout the latest release. You can just use the master branch but it can be unstable.
 ```
 go get -u https://github.com/kaakaa/matterpoll-emoji
 cd $GOPATH/src/github.com/kaakaa/matterpoll-emoji
+git co $(git describe --tags)
```
Comment (kaakaa) [inline README.md]: `git co` is default command?

Comment (Hanzei) [reply README.md]: You`r correct! Fixed that

File: README.md [hunk 2]
```diff
@@ -18,25 +18,54 @@ Polling feature for Mattermost's custom slash command.
...
 
 Copy the Token from your newly created slash command
 
 ### Setup server
 
-Clone this repository
+#### Run pre compiled release
+
+Download the latest version at https://github.com/kaakaa/matterpoll-emoji/releases/latest.
+Decompress it and change parameter in `config.json` as you need them
```
[the remark this answers is not in the document]
Comment (Hanzei) [reply README.md]: Can we change the behaviour of the build script to include `config.json` not `.config.json`?

[the remark this answers is not in the document]
Comment (kaakaa) [reply README.md]: Yes we can. that's a nice way!

[the remark this answers is not in the document]
Comment (Hanzei) [reply README.md]: @kaakaa Can you take care of this? Makefile are still voodoo magic for me

[the remark this answers is not in the document]
Comment (kaakaa) [reply README.md]: Okay, I'll commit this fixing soon.

**Pull request merged**